src/frontend/src/components/Header/MobileHeader.jsx

```diff
@@ -44,13 +44,13 @@ const useStyles = makeStyles((theme) => ({
     width: 250,
   },
   paper: {
-    background: '#242424',
+    backgroundColor: '#00233a',
```

Comment:
`backgroundColor: theme.palette.primary.main`

Reply:
I think I had issues with this one, where it wouldn't accept the theme, but I can try again. It's possible I had the wrong syntax